.github/ISSUE_TEMPLATE/release-runtime.md

```diff
@@ -30,6 +30,7 @@ candidate branch or started an additional release candidate branch (rc-2, rc-3,
 - [ ] Verify [new migrations](https://github.com/paritytech/cumulus/blob/master/docs/release.md#new-migrations) complete successfully, and the
     runtime state is correctly updated for any public (non-private/test)
     networks
+- [ ] Verify new extrinsics have been correctly whitelisted/blacklisted for [proxy filters](https://github.com/paritytech/cumulus/blob/master/docs/release.md#proxy-filtering).
```

Comment:
@NachoPal 
there is no mention about proxy filters here https://github.com/paritytech/cumulus/blob/master/docs/release.md#proxy-filtering